awx/ui_next/src/components/StatusIcon/StatusIcon.jsx

```diff
@@ -93,42 +93,43 @@ SkippedBottom.displayName = 'SkippedBottom';
 
 const StatusIcon = ({ status, ...props }) => {
   return (
-    <div {...props} data-job-status={status}>
+    <div {...props} data-job-status={status} aria-label={status}>
       {status === 'running' && <RunningJob />}
       {(status === 'new' ||
         status === 'pending' ||
         status === 'waiting' ||
         status === 'never updated') && <WaitingJob />}
       {(status === 'failed' || status === 'error' || status === 'canceled') && (
```

Comment:
(more an open question than anything) I'm wondering if `RunningJob` and `WaitingJob` need `aria-hidden="true"` as well? Based on my reading of https://pf4.patternfly.org/accessibility-guide/#icons I think they might?